src/unit_tests/shared/test_version_op.c

```diff
@@ -160,7 +308,6 @@ void test_get_unix_version_opensuse_tumbleweed(void **state)
     assert_string_equal(ret->os_version, "");
     assert_string_equal(ret->os_platform, "opensuse-tumbleweed");
     assert_string_equal(ret->sysname, "Linux");
-    assert_string_equal(ret->os_build, "rolling");
```

Comment:
We've removed the `rolling` word but this doesn't seem to affect the final result (at least in my Manjaro).
Do you know why?

Reply:
After the discussion we have, I only see that assignment for internal use and then the variable is freed, nothing about "rolling" is present in the global db, that's why I thought it was not strictly necessary to use that step in between.

Reply:
Ok, thank you!